lib/system/supports-colors.js

```diff
@@ -49,12 +45,16 @@ module.exports = (function () {
     return true;
   }
 
-  if (process.env.TERM === 'dumb') {
+  if (/^screen|^xterm|^vt100|color|ansi|cygwin|linux/i.test(process.env.TERM)) {
+    return true;
+  }
+
+  if (process.stdout && !process.stdout.isTTY) {
     return false;
   }
 
-  if (/^screen|^xterm|^vt100|color|ansi|cygwin|linux/i.test(process.env.TERM)) {
-    return true;
+  if (process.env.TERM === 'dumb') {
```

Comment:
I think this was done on purpose since

 if (/^screen|^xterm|^vt100|color|ansi|cygwin|linux/i.test(process.env.TERM)) { 

is moved to below if ('COLORTERM' in process.env) {

and moved ontop of

if (process.stdout && !process.stdout.isTTY) {

But this change does something since it now works in windows, and Linux.